src/azure-cli/azure/cli/command_modules/cdn/tests/latest/recordings/test_cdn_custom_domain_https_msft.yaml

```diff
@@ -0,0 +1,2323 @@
+    status:
+      code: 200
+      message: OK
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - cdn custom-domain show
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - -g -n --endpoint-name --profile-name
+      User-Agent:
+      - python/3.7.7 (Darwin-19.4.0-x86_64-i386-64bit) msrest/0.6.10 msrest_azure/0.6.3
+        azure-mgmt-cdn/4.1.0rc1 Azure-SDK-For-Python AZURECLI/2.4.0
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Cdn/profiles/profile123/endpoints/cdn-cli-test-4/customDomains/customdomain000002?api-version=2019-06-15-preview
+  response:
+    body:
+      string: "{\r\n  \"name\":\"customdomain000002\",\"id\":\"/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001/providers/Microsoft.Cdn/profiles/profile123/endpoints/cdn-cli-test-4/customdomains/customdomain000002\"\
+        ,\"type\":\"Microsoft.Cdn/profiles/endpoints/customdomains\",\"properties\"\
+        :{\r\n    \"provisioningState\":\"Succeeded\",\"resourceState\":\"Active\"\
+        ,\"hostName\":\"customdomain000002.cdn-cli-test-4.azfdtest.xyz\",\"validationData\"\
+        :null,\"customHttpsProvisioningState\":\"Disabled\",\"customHttpsProvisioningSubstate\"\
+        :\"None\",\"customHttpsParameters\":null\r\n  }\r\n}"
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '625'
+      content-type:
+      - application/json; odata.metadata=minimal
+      date:
+      - Mon, 27 Apr 2020 21:59:23 GMT
+      expires:
+      - '-1'
+      odata-version:
+      - '4.0'
+      pragma:
+      - no-cache
+      server:
+      - Microsoft-IIS/8.5
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      transfer-encoding:
+      - chunked
+      vary:
+      - Accept-Encoding
+      x-aspnet-version:
+      - 4.0.30319
+      x-content-type-options:
+      - nosniff
+      x-powered-by:
+      - ASP.NET
+    status:
+      code: 200
+      message: OK
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - cdn custom-domain enable-https
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - -g -n --endpoint-name --profile-name --min-tls-version
+      User-Agent:
+      - python/3.7.7 (Darwin-19.4.0-x86_64-i386-64bit) msrest/0.6.10 msrest_azure/0.6.3
+        azure-mgmt-cdn/4.1.0rc1 Azure-SDK-For-Python AZURECLI/2.4.0
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Cdn/profiles/profile123?api-version=2019-06-15-preview
+  response:
```

Comment:
Same here. This is for a pull request to fix this call and the tests seem off. It's weird.

Reply:
Again, the tests must perform required setup before testing the actual area of interest.

Reply:
Thanks for the explanation. For some reason I assumed these were combined with the other tests considering the setup could be handled in that manor. But this makes it a lot clearer.

My apologies for the comments going a bit to far. The lock-down has put me on edge and frustration got the better of me.